pandas/tests/io/test_parquet.py

```diff
@@ -376,6 +376,11 @@ def test_columns_dtypes_invalid(self, engine):
         ]
         self.check_error_on_write(df, engine, ValueError)
 
+    def test_to_bytes_without_path_or_buf_provided(self):
+        # GH 37105
+        df = pd.DataFrame()
+        df.to_parquet()
```

Comment:
check one of the test dataframes, and test that the round-trip works as well, alt you can write it to a file and check that its the ssame.

Reply:
Done